### PR TITLE
[EDU-3009] refactor: modified support tickets and BES

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Changed wording in BES
- Added ticket types
- Added definition on ticket type severities

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-3009]: https://aziontech.atlassian.net/browse/EDU-3009?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx [hunk 1]
```diff
@@ -45,21 +45,56 @@ Para obter informações mais detalhadas, consulte as opções de [Planos de Ser
...
 | Production system down | Você não tem como contornar o problema e seu negócio está significativamente impactado. Funções importantes de sua aplicação estão indisponíveis. |
 | Business-critical system down | Você não tem como contornar o problema e seu negócio está em risco. Funções críticas de sua aplicação estão indisponíveis. |
 
-### Gravidade dos casos e tempos de resposta de acordo com os planos de serviço
+#### Request
+
+A tabela abaixo descreve as severidades para tickets do tipo **Request**:
+
+| Severidade | Definição |
+| -------------------- | --------- |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx:80]:
```suggestion
| Severidade | Definição |
| ----------- | --------- |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx [hunk 2]
```diff
@@ -45,21 +45,56 @@ Para obter informações mais detalhadas, consulte as opções de [Planos de Ser
...
+| CNAMEs already in use | Você encontrou o erro *CNAMEs already in use* ao listar um CNAME em seu [domínio Azion](/pt-br/documentacao/produtos/edge-application/domains/). |
+| Integration Services | Você deseja solicitar [Integration Services](/pt-br/documentacao/servicos/solutions-lab/integration-services/) para migrar seu conteúdo para a Azion. |
+| Template (Preview) | Você precisa que um [Template](/pt-br/documentacao/produtos/marketplace/templates/) seja adicionado ao Azion Marketplace. |
+
+#### Question
+
+A tabela abaixo descreve as severidades para tickets do tipo **Question**:
+
+| Severidade | Definição |
+| -------------------- | --------- |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx:93]:
```suggestion
| Severidade | Definição |
| ----------- | --------- |
```
```

File: src/content/docs/en/pages/main-menu/reference/services/support/business-events-support.mdx [hunk 3]
```diff
@@ -18,7 +18,7 @@ This service offers a follow-up from expert engineers, consultancy, and the nece
 - Continuous monitoring of system performance during the event.
 - Dedicated technical assistance and support to address any emerging issue.
 
-If you have an **Enterprise** or **Mission Critical** Service Plan, you can request this service through the [Support Portal](https://tickets.azion.com/en/support/login/new), within 72 hours prior to the event.
+If you have an **Enterprise** or **Mission Critical** Service Plan, you can request this service through the [Support Portal](https://tickets.azion.com/en/support/login/new), within at least 72 hours prior to the event.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/services/support/business-events-support.mdx:21]:
```suggestion
If you have an **Enterprise** or **Mission Critical** Service Plan, you can request this service through the [Support Portal](https://tickets.azion.com/en/support/login/new), within at least `72 hours` prior to the event.
```

Comment (hannah-sk) [reply src/content/docs/en/pages/main-menu/reference/services/support/business-events-support.mdx:21]: to match PT

File: src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx [hunk 4]
```diff
@@ -45,21 +45,56 @@ For more detailed information, see [Azion Service Plans](https://www.azion.com/e
...
+The goal is to provide an appropriate response time to each case based on the impact on your business. Opening tickets with the correct severity level will help the team get you in touch with the appropriate support professionals and within the estimated response times.
+
+#### Incident
 
-To understand how Azion regards the severity of tickets, see the table:
+The table below describes the severity options for tickets of the **Incident** type.
 
-| Ticket Severity | Definition |
+| Severity | Definition |
 | --------------- | ---------- |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx:69]:
```suggestion
| Severity | Definition |
| -------- | ---------- |
```

File: src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx [hunk 5]
```diff
@@ -45,21 +45,56 @@ For more detailed information, see [Azion Service Plans](https://www.azion.com/e
...
+| CNAMEs already in use | You have encountered the error *CNAMEs already in use* when listing a CNAME in one of your [Azion domains](/en/documentation/products/edge-application/domains/). |
+| Integration Services | You want to request [Integration Services](/en/documentation/services/solutions-lab/integration-services/) to migrate your content to Azion. |
+| Template (Preview) | You need a new [Template](/en/documentation/products/marketplace/templates/) to be added to the Azion Marketplace. |
+
+#### Question
+
+The table below describes the severity options for tickets of the **Question** type.
+
+| Severity | Definition |
+| --------------- | ---------- |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx:93]:
```suggestion
| Severity | Definition |
| -------- | ---------- |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx [hunk 6]
```diff
@@ -45,21 +45,56 @@ Para obter informações mais detalhadas, consulte as opções de [Planos de Ser
...
+O objetivo é fornecer um tempo de resposta adequado para cada caso com base no impacto para o seu negócio. A abertura de tickets com o nível de severidade correto ajudará a Azion a colocar você em contato com os profissionais de suporte adequados dentro dos tempos de resposta estimados.
+
+#### Incident
 
-Para entender como a Azion considera a severidade dos ticket, veja a tabela:
+A tabela abaixo descreve as severidades para tickets do tipo **Incident**:
 
-| Severidade do Ticket | Definição |
+| Severidade | Definição |
 | -------------------- | --------- |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx:69]:
```suggestion
| Severidade | Definição |
| ----------- | --------- |
```

**Review by hannah-sk — APPROVED** [review]
deixei algumas sugestões (basicamente pro mdx ficar mais organizadinho)

File: src/content/docs/en/pages/main-menu/reference/services/support/business-events-support.mdx [hunk 7]
```diff
@@ -8,7 +8,7 @@ permalink: /documentation/services/business-events-support/
 
 **Business Events Support** refers to the support and assistance provided to assist customers during events, in real time. In this service, several tests are performed to assess the performance and capacity of the system under conditions of intense load.
 
-This service offers a follow-up from expert engineers, consultancy, and the necessary support to maximize the probability of success and remove obstacles during the event. Azion specialists conduct a highly focused engagement to provide guidance during the operation, as well as real-time support, at any time and lasting up to 6 hours of follow-up. 
+This service offers a follow-up, assistance, and the necessary support from expert engineers to maximize the probability of success and remove obstacles during the event. Azion specialists conduct a highly focused engagement to provide guidance during the operation, as well as real-time support, at any time and lasting up to 6 hours of follow-up.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/services/support/business-events-support.mdx:11]:
```suggestion
This service offers follow-up, assistance, and the necessary support from expert engineers to maximize the probability of success and remove obstacles during the event. Azion specialists conduct a highly focused engagement to provide guidance during the operation, as well as real-time support, at any time and lasting up to `6 hours` of follow-up. 
```

File: src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx [hunk 8]
```diff
@@ -45,21 +45,56 @@ For more detailed information, see [Azion Service Plans](https://www.azion.com/e
...
+
+:::tip
+If you want to open a ticket, see the guide on [How to open a Support Ticket](/en/documentation/products/guides/open-tickets/).
+:::
+
+There are three options available to categorize tickets into types:
+
+- **Incident**: to report an incident, such as system failures or errors that require immediate attention.
+- **Request**: to make a specific request, such as integrations, solutions, or new features.
+- **Question**: to clarify questions or concerns regarding the Azion Platform.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx:58]:
```suggestion
- **Question**: to clarify questions or concerns regarding the Azion's platform.
```

File: src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx [hunk 9]
```diff
@@ -45,21 +45,56 @@ For more detailed information, see [Azion Service Plans](https://www.azion.com/e
...
 | Business-critical system down | You have no way around the problem and your business is at risk. Critical functions of your application are unavailable. |
 
-### Severity of cases and response times according to service plans
+#### Request
+
+The table below describes the severity options for tickets of the **Request** type.
+
+| Severity | Definition |
+| --------------- | ---------- |
+| General Guidance | You have a general request for the Azion Platform. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx:81]:
```suggestion
| General Guidance | You have a general request for the Azion Edge Platform. |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/business-support-events.mdx [hunk 10]
```diff
@@ -8,7 +8,7 @@ permalink: /documentacao/servicos/business-events-support/
 
 **Business Events Support** refere-se ao suporte e à assistência prestados para auxiliar os clientes durante os eventos em tempo real. Neste serviço, vários testes são realizados para avaliar o desempenho e a capacidade do sistema em condições de carga intensa.
 
-Este serviço oferece um acompanhamento de engenheiros especialistas, consultoria e o suporte necessário para maximizar a probabilidade de sucesso e remover obstáculos durante o evento. Os especialistas da Azion realizam um engajamento altamente focado para fornecer orientação durante a operação, bem como suporte em tempo real, a qualquer momento e com duração de até 6 horas de acompanhamento.
+Este serviço oferece um acompanhamento de engenheiros especialistas, assistência e o suporte necessário para maximizar a probabilidade de sucesso e remover obstáculos durante o evento. Os especialistas da Azion realizam um engajamento altamente focado para fornecer orientação durante a operação, bem como suporte em tempo real, a qualquer momento e com duração de até 6 horas de acompanhamento.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/business-support-events.mdx:11]:
```suggestion
Este serviço oferece um acompanhamento de engenheiros especialistas, assistência e o suporte necessário para maximizar a probabilidade de sucesso e remover obstáculos durante o evento. Os especialistas da Azion realizam um engajamento altamente focado para fornecer orientação durante a operação, bem como suporte em tempo real, a qualquer momento e com duração de até `6 horas` de acompanhamento.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx [hunk 11]
```diff
@@ -45,21 +45,56 @@ Para obter informações mais detalhadas, consulte as opções de [Planos de Ser
...
 | Business-critical system down | Você não tem como contornar o problema e seu negócio está em risco. Funções críticas de sua aplicação estão indisponíveis. |
 
-### Gravidade dos casos e tempos de resposta de acordo com os planos de serviço
+#### Request
+
+A tabela abaixo descreve as severidades para tickets do tipo **Request**:
+
+| Severidade | Definição |
+| -------------------- | --------- |
+| General Guidance | Você tem uma solicitação para a Plataforma da Azion. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx:81]:
```suggestion
| General Guidance | Você tem uma solicitação para a plataforma da Azion. |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx [hunk 12]
```diff
@@ -45,21 +45,56 @@ Para obter informações mais detalhadas, consulte as opções de [Planos de Ser
...
+| Integration Services | Você deseja solicitar [Integration Services](/pt-br/documentacao/servicos/solutions-lab/integration-services/) para migrar seu conteúdo para a Azion. |
+| Template (Preview) | Você precisa que um [Template](/pt-br/documentacao/produtos/marketplace/templates/) seja adicionado ao Azion Marketplace. |
+
+#### Question
+
+A tabela abaixo descreve as severidades para tickets do tipo **Question**:
+
+| Severidade | Definição |
+| -------------------- | --------- |
+| General Guidance | Você tem uma dúvida sobre a Plataforma da Azion. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx:94]:
```suggestion
| General Guidance | Você tem uma dúvida sobre a plataforma da Azion. |
```

**Review by MarianaAguilera — APPROVED** [review]
Left a couple of suggestions. Approved with @hannah-sk's comments.

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx [hunk 13]
```diff
@@ -45,21 +45,56 @@ Para obter informações mais detalhadas, consulte as opções de [Planos de Ser
...
-### Gravidade dos casos e tempos de resposta de acordo com os planos de serviço
+#### Request
+
+A tabela abaixo descreve as severidades para tickets do tipo **Request**:
+
+| Severidade | Definição |
+| ----------- | --------- |
+| General Guidance | Você tem uma solicitação para a plataforma da Azion. |
+| New Feature | Você deseja solicitar uma nova funcionalidade. |
+| Solutions Labs | Você deseja solicitar a assistência de um especilista. |
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx:83]:
```suggestion
| Solutions Labs | Você deseja solicitar a assistência de um especialista para propor uma nova solução. |
```

File: src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx [hunk 14]
```diff
@@ -45,21 +45,56 @@ For more detailed information, see [Azion Service Plans](https://www.azion.com/e
...
-### Severity of cases and response times according to service plans
+#### Request
+
+The table below describes the severity options for tickets of the **Request** type.
+
+| Severity | Definition |
+| -------- | ---------- |
+| General Guidance | You have a general request for the Azion Edge Platform. |
+| New Feature | You want to request a feature. |
+| Solutions Labs | You want to request the assistance of a specialist. |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx:83]:
```suggestion
| Solutions Labs | You want to request the assistance of a specialist to propose a new solution. |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx [hunk 15]
```diff
@@ -45,21 +45,56 @@ Para obter informações mais detalhadas, consulte as opções de [Planos de Ser
...
+
+A tabela abaixo descreve as severidades para tickets do tipo **Request**:
+
+| Severidade | Definição |
+| ----------- | --------- |
+| General Guidance | Você tem uma solicitação para a plataforma da Azion. |
+| New Feature | Você deseja solicitar uma nova funcionalidade. |
+| Solutions Labs | Você deseja solicitar a assistência de um especialista para propor uma nova solução. |
+| CNAMEs already in use | Você encontrou o erro *CNAMEs already in use* ao listar um CNAME em seu [domínio Azion](/pt-br/documentacao/produtos/edge-application/domains/). |
+| Integration Services | Você deseja solicitar [Integration Services](/pt-br/documentacao/servicos/solutions-lab/integration-services/) para migrar seu conteúdo para a Azion. |
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/technical-support.mdx:85]:
```suggestion
| Integration Services | Você deseja solicitar [Integration Services](/pt-br/documentacao/servicos/solutions-lab/integration-services/) para realizar uma nova integração com a Azion. |
```

File: src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx [hunk 16]
```diff
@@ -45,21 +45,56 @@ For more detailed information, see [Azion Service Plans](https://www.azion.com/e
...
+
+The table below describes the severity options for tickets of the **Request** type.
+
+| Severity | Definition |
+| -------- | ---------- |
+| General Guidance | You have a general request for the Azion Edge Platform. |
+| New Feature | You want to request a feature. |
+| Solutions Labs | You want to request the assistance of a specialist to propose a new solution. |
+| CNAMEs already in use | You have encountered the error *CNAMEs already in use* when listing a CNAME in one of your [Azion domains](/en/documentation/products/edge-application/domains/). |
+| Integration Services | You want to request [Integration Services](/en/documentation/services/solutions-lab/integration-services/) to migrate your content to Azion. |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/services/support/technical-support.mdx:85]:
```suggestion
| Integration Services | You want to request [Integration Services](/en/documentation/services/solutions-lab/integration-services/) initiate a new integration with Azion. |
```

**Pull request merged**